config/initializers/cypress_rails.rb

```diff
@@ -22,7 +25,7 @@
 CypressRails.hooks.before_server_stop do
   # Called once, at_exit
   puts "Cleaning up and stopping server for end to end tests."
-  Rake::Task["db:truncate_all"].invoke
+  Rake::Task["db:truncate_all"].invoke if ENV["SKIP_SEED_DATA"].blank?
```

Comment:
Only truncate DB if we seeded data.

Reply:
I don't see this rake task defined in the project 
https://gist.github.com/djuber/15c5d6c865c9e96304bfbf7a09b27054

There are hints on stack overflow on creating a task like that (to truncate) - but it's unclear why or whether it was working before (unless it's only defined by cypress rails).

In running cypress locally (via the e2e-ci script) I needed to manually reload the database schema between runs, suggesting we weren't truncating correctly before, either.

db:schema:load might do just what you wanted, it drops tables and recreates them (it has the added cost of re-running migrations relative to truncation, but will leave the db in a clean state after execution).

Reply:
I got it sorted out. See https://github.com/forem/forem/pull/14285/commits/bd05b605c2c8494bc9c5921f95b7901ab61a25b2

Reply:
yes, I see it now (it's in [active_record](https://github.com/rails/rails/blob/main/activerecord/lib/active_record/railties/databases.rake#L77-L80)) - I wonder why it wasn't shown as an available task by rake (but that's not at all important here).

Reply:
@djuber Only tasks with a description will be shown by `rake -T` and friends and that's been commented out for this tasks. You can still see it if you use the `--all`/`-A` switch.

![image](https://user-images.githubusercontent.com/47985/126585236-e29be313-3ba2-4501-bcfc-b5a5d6dfb80b.png)